srtcore/udt.h

```diff
@@ -115,10 +115,6 @@ modified by
 // This facility is used only for select() function.
 // This is considered obsolete and the epoll() functionality rather should be used.
 typedef std::set<SRTSOCKET> ud_set;
-#define UD_CLR(u, uset) ((uset)->erase(u))
-#define UD_ISSET(u, uset) ((uset)->find(u) != (uset)->end())
-#define UD_SET(u, uset) ((uset)->insert(u))
-#define UD_ZERO(uset) ((uset)->clear())
 #endif
```

Comment:
Accidental removal? Better keep changes strictly related to the topic. I think these can be attached to the PR that makes a revolution in epoll.

Reply:
Those are unused. Let's move them to epoll PR.